src/crimson/osd/object_context.h

```diff
@@ -121,7 +121,7 @@ class ObjectContext : public ceph::common::intrusive_lru_base<
   bool recovery_read_marker = false;
 
   template <typename Lock, typename Func>
-  auto _with_lock(Lock&& lock, Func&& func) {
+  auto _with_lock(Lock &lock, Func &&func) {
```

Comment:
i understand that we've been putting `&` near the variable. but `&` is part of the type, see https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines for more examples, so..